tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p> </p><blockquote> <strong>Note:</strong> All the following methods in this tutorial are explained using the <code>Estimator</code> class to evaluate the gradients, but, in an analogous way to the Parameter Shift gradients just introduced, the method explanation can also be applied to <code>Sampler</code>-based gradients. Both versions are available in <code>algorithms.gradients</code>.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>